src/main/java/com/carleton/CapstoneSystem/Program.java

```diff
@@ -0,0 +1,35 @@
+package com.carleton.CapstoneSystem;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Program {

```

Comment:
do you think it would be better if we make this an enum since we know all the programs in Carleton.

Reply:
i honestly thing it would be the same